### PR TITLE
Some language improvements and better active voice for OTEL collector.

### PR DESCRIPTION
**Pull request by jhkrug**
As per title.

### CODE REVIEW

File: docs/howtos/telemetry/10-opentelemetry-qs.md [hunk 1]
```diff
@@ -95,20 +94,20 @@ $ minikube addons enable ingress
...
-with Cert Manager, [see the compat chart](https://github.com/open-telemetry/opentelemetry-operator#opentelemetry-operator-vs-kubernetes-vs-cert-manager).
+At the time of writing (2024-12-23), only specific versions of OpenTelemetry are compatible
+with Cert Manager, [see the compatibility chart](https://github.com/open-telemetry/opentelemetry-operator#opentelemetry-operator-vs-kubernetes-vs-cert-manager).
 
-We will install the latest cert-manager Helm chart:
+You should install the latest cert-manager Helm chart:
 
 :::note
-At time of writing the latest cert-manager chart version is `v1.15.1`
+At time of writing (2024-12-23) the latest cert-manager chart version is `v1.15.1`
```
Comment (jvanz) [inline docs/howtos/telemetry/10-opentelemetry-qs.md:110]:
```suggestion
At time of writing (2024-07-17) the latest cert-manager chart version is `v1.15.1`
```

File: docs/howtos/telemetry/10-opentelemetry-qs.md [hunk 2]
```diff
@@ -95,20 +94,20 @@ $ minikube addons enable ingress
...
+into the PolicyServer pod.
 
-The OpenTelemetry Operator requires [cert-manager](https://cert-manager.io/docs/installation/)
-to be installed inside of the cluster.
+The OpenTelemetry Operator requires installation of [cert-manager](https://cert-manager.io/docs/installation/)
+in the cluster.
 
-At the time of writing, only specific versions of OpenTelemetry are compatible
-with Cert Manager, [see the compat chart](https://github.com/open-telemetry/opentelemetry-operator#opentelemetry-operator-vs-kubernetes-vs-cert-manager).
+At the time of writing (2024-12-23), only specific versions of OpenTelemetry are compatible
```
Comment (jvanz) [inline docs/howtos/telemetry/10-opentelemetry-qs.md:104]:
```suggestion
At the time of writing (2022-06-21), only specific versions of OpenTelemetry are compatible
```

File: docs/howtos/telemetry/20-tracing-qs.md [hunk 3]
```diff
@@ -12,36 +12,37 @@ doc-topic: [operator-manual, telemetry, tracing, quick-start]
...
-to manage all the different Jaeger components. The operator can be installed in many ways. We are going to install via Helm charts.
+You use the [Jaeger Operator](https://github.com/jaegertracing/jaeger-operator)
+to manage the different Jaeger components. You can install the Jaeger Operator using Helm charts.
 
-At the time of writing, only specific versions of Jaeger are compatible with
-Cert Manager, [see the compat chart](https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger-operator/COMPATIBILITY.md).
+:::note
 
-To install the Helm chart:
+At the time of writing (2024-12-23), only specific versions of Jaeger are compatible with
```
Comment (jvanz) [inline docs/howtos/telemetry/20-tracing-qs.md:37]:
```suggestion
At the time of writing (2022-06-21), only specific versions of Jaeger are compatible with
```

File: docs/howtos/telemetry/30-metrics-qs.md [hunk 4]
```diff
@@ -69,10 +69,10 @@ prometheus:
 EOF
 ```
 
-Let's install the Prometheus stack Helm Chart:
+You should install the Prometheus stack Helm chart:
 
 :::note
-At time of writing the latest chart version is `51.5.3`
+At time of writing (2024-12-23) the latest chart version is `51.5.3`
```
Comment (jvanz) [inline docs/howtos/telemetry/30-metrics-qs.md:75]:
```suggestion
At time of writing (2023-11-17) the latest chart version is `51.5.3`
```

**Review by jvanz — CHANGES_REQUESTED** [review]
Thanks for the enhancements! I've only fixed some date to reflect the actual time when that piece of text has been written. Maybe we should create a tech-debt to double check that sentences or update them with more recent information.

**Comment by jhkrug** [comment]
Nope, all good. Thanks.

**Pull request merged**